### DISCUSSION TITLE
Как лучше поднимать доку и тестировать изменения в библиотеке?

### DISCUSSION THREAD

**Discussion by scffs** [VKCOM/VKUI#5215] [Q&A] [answered]
Сейчас я локально поднимаю доку через `yarn docs:styleguide` (хотя в `scripts` этой команды нет, только [тут](https://github.com/VKCOM/VKUI/blob/master/docs/CONTRIBUTING.md)), а изменения смотрю через `Readme.md` необходимого компонента (-ов), это единственный вариант или есть что-то лучше? Можно по сути через сторибук ещё

**Comment by SevereCloud** [comment, marked as answer]
> в `scripts` этой команды нет, только [тут](https://github.com/VKCOM/VKUI/blob/master/docs/CONTRIBUTING.md)

Вот она 

https://github.com/VKCOM/VKUI/blob/02d353bb71355c0247f4011dd16aa4cbb77f2d82/package.json#L103-L105

> Можно по сути через сторибук ещё

Да, можно тестировать через сторибук, используя команду `yarn docs:storybook`

**Reply by scffs** [reply]
А, я просто в другой package смотрел, сорри
